### ISSUE TITLE
failing to upload large file to Backblaze B2

### ISSUE THREAD

**Issue by jtbayly** [watchfulli/XCloner-Wordpress#14]
Hi there,

I've got full site backups that are about 2GB in size, and although a scheduled backup of just the DB (about 30 MB) uploads to to B2 just fine, I have yet to get a full size backup to upload properly.

On a scheduled backup, the emailed error is as follows:
"<>Error Message: Received error from B2: c001_v0001093_t0044 is too busy"

I also tried to manually upload the backup file, and the UI eventually shows the following error: 
"errorHTTP/2.0 404"

The log contains the following after the manual attempt to upload:

`[2017-10-12 18:51:10] xcloner_file_system.INFO: Cleaning the backup storage on matching rules [] []
[2017-10-12 18:49:17] xcloner_remote_storage.INFO: Transferring backup *******.com-2017-10-12_18-23-sql-a5f55.tar to remote storage BACKBLAZE [""] []
[2017-10-12 18:49:16] xcloner_remote_storage.INFO: Doing BACKBLAZE remote storage cleanup for 180 days limit [] []
[2017-10-12 18:49:15] xcloner_remote_storage.INFO: Creating the BACKBLAZE remote storage connection [""] []`

Any ideas?

**Comment by ovidiul** [comment]
The issue seems to come from Backblaze, you might need to check with their support why you get that error

**Comment by jtbayly** [comment]
I've been in touch with Backblaze support, and they said that the "too busy" error could have been them (just yesterday).

However, today I'm not getting that error, but it's still not uploading to Backblaze, either automatically or manually. 

In the UI, I get the same "errorHTTP/2.0 404" on manual upload attempt after waiting for a while. However, there is no indication in the Xcloner logs of any sort of failure, for either the auto or manual upload. At the very least, I believe there is a problem with the xCloner logging, since I can't find any indication that something is going wrong, much less get any helpful data about what is happening.

Has anybody been able to backup a multi-gigabyte file to Backblaze B2 with this plugin? Just trying to narrow things down.

Thanks for your work.

**Comment by jtbayly** [comment]
Just to update, Backblaze is offering to investigate further on their end, but they requested logs from today's failure, and I can't provide them anything because there is no failure recorded in the logs.

**Comment by jtbayly** [comment]
At this point, the problem is clearly with Xcloner, not Backblaze. 

I've lowered the Split Backup Archive Limit to 1024 and tried again. The CSV file uploads appropriately, but the .tar backup files don't transfer, and I get no email about the backup succeeding or failing. No indication of failure in the logs. I've also attempted switching to a new Application Key, and it made no difference.

An incremental backup of 127MB works fine, though.
